pegasus/sites.v3/code.org/public/js/angular-selectize.js

```diff
@@ -0,0 +1,257 @@
+/**
+ * Directive to convert a select into a selectize.js hybrid textbox and <select>
+ * Supports an ngOptions expression. Tested with:
+ *  `label for value in array`
+ *  `select as label for value in array`
+ * In theory supports the same options as selectize.js
+ *
+ * Usage:
+ * <select
+ *   multiple
+ *   ng-model="selected"
+ *   ng-options="option.id as option.name for option in options"
+ *   selectize="{ plugins: ['remove_button'], create: 'true' }">
+ * </select>
+ *
+ * Attributes:
+ *   multiple: Converts the select into text input of tags
+ *
+ * (c) 2014 Evan Oxfeld https://github.com/EvanOxfeld/angular-selectize.js
+ * License: MIT
+**/
```

Comment:
It would be a good idea to eventually split out vendor assets from `/pegasus/sites.v3/code.org/public/js`, but this existed prior to this change so not blocking.  Any reason this shouldn't be in `/shared/js`?